src/tqec/detectors/operation.py

```diff
@@ -0,0 +1,244 @@
+from typing import Any
+from dataclasses import dataclass
+
+import cirq
+
+STIM_TAG = "STIM_OPERATION"
+
+
+class ShiftCoords(cirq.Operation):
+    def __init__(self, *shift: int) -> None:
+        """
+        Annotates that the qubit/detector coordinate origin is being moved.
+
+        This is a replication of the [stimcirq.ShiftCoordsAnnotation](https://github.com/quantumlib/Stim/blob/main/glue/cirq/stimcirq/_shift_coords_annotation.py)
+        class. We can directly use `stimcirq.ShiftCoordsAnnotation` here, however,
+        replication brings the class into the `tqec` namespace and is useful for the
+        potential future iteration.
+
+        :param shift: How much to shift each coordinate.
+        """
+        self._shift = shift
+
+    @property
+    def shift(self) -> tuple[int, ...]:
+        """The shift the operation represents."""
+        return self._shift
```

Comment:
```suggestion
        :param shifts: How much to shift each coordinate.
        """
        self._shifts = shifts

    @property
    def shifts(self) -> tuple[int, ...]:
        """The shifts the operation represents."""
        return self._shifts
```
see previous comment.